pom.xml

```diff
@@ -25,8 +24,11 @@
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 		<keycloak.version>4.8.0.Final</keycloak.version>
-		<jenkins.version>1.625.3</jenkins.version>
-		<jenkins-test-harness.version>2.13</jenkins-test-harness.version>
+		<jenkins.version>2.60.3</jenkins.version>
```

Comment:
Increasing `jenkins.version` to 2.184 removes all implied plugins. Non of the currently implied plugins on the plugin page like `Trilead API` is used by keycloak-plugin.

Reply:
2.184 isn't lts. so maybe 2.190.1? I just wanted to do a huuuge jump in this PR, but its probably not a bad idea

Reply:
Yea lets go up to 2.190.1 since I'm suspecting either @devlauer is unavailable or moved on from this plugin. We can always downgrade later.

I personally decided to go with https://plugins.jenkins.io/oic-auth/